### PR TITLE
Web template strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**
A lot of things still need explaining and clarification, but this deals with a preliminary set of things :)

### CODE REVIEW

File: src/web/web_templates.ml [hunk 1]
```diff
@@ -3762,7 +3762,7 @@ let booth () =
       div ~a:[a_style "text-align:center; margin-bottom:20px;"] [
         span ~a:[a_id "progress1"; a_style "font-weight:bold;"] [txt (s_ "Input credential")];
         txt " — ";
-        span ~a:[a_id "progress2"] [txt (s_ "Answer to questions")];
+        span ~a:[a_id "progress2"] [txt (s_ "Answers to questions")];
```
Comment (comradekingu) [inline src/web/web_templates.ml:3765]: Or?

File: src/web/web_templates.ml [hunk 2]
```diff
@@ -2509,7 +2509,7 @@ let election_admin ?shuffle_token ?tally_token election metadata state get_token
                  ~value:(s_ "Proceed to vote counting")
                  string;
               txt " ";
-              txt (s_ "Warning: this action is irreversible; the election will be definitively closed.");
+              txt (s_ "Warning: This action is irreversible; the election definitively be closed.");
```
Comment (glondu) [inline src/web/web_templates.ml:2512]: This feels a bit odd without the "will", is it intentionnal?

File: src/web/web_templates.ml (hunk 1 above)
Comment (glondu) [reply src/web/web_templates.ml:3765]: I would keep "Answer to questions" here, as all other items in the breadcrumb are verbs in infinitive form. Or all items should be switched to nouns.

**Review by glondu — CHANGES_REQUESTED** [review]

File: src/web/web_templates.ml (hunk 2 above)
Comment (comradekingu) [reply src/web/web_templates.ml:2512]: Good catch :)

Comment (comradekingu) [inline src/web/web_templates.ml:2512]:
```suggestion
              txt (s_ "Warning: This action is irreversible; the election will be definitively closed.");
```

File: src/web/web_templates.ml (hunk 1 above)
Comment (comradekingu) [reply src/web/web_templates.ml:3765]: For another day.

Comment (comradekingu) [inline src/web/web_templates.ml:3765]:
```suggestion
        span ~a:[a_id "progress2"] [txt (s_ "Answer to questions")];
```

**Pull request merged**